patches/v8/0019-Implement-cross-request-context-promise-resolve-hand.patch

```diff
@@ -1,27 +1,11 @@
-From 8cf4fe0f72fbaf25028ebd152559bd363da6599e Mon Sep 17 00:00:00 2001
+From 7724507f0525deef0a484d33d2efee3586728520 Mon Sep 17 00:00:00 2001
 From: James M Snell <jasnell@gmail.com>
 Date: Mon, 16 Sep 2024 09:56:04 -0700
-Subject: [PATCH] Implement cross-request context promise resolve handling
+Subject: Implement cross-request context promise resolve handling
 
----
```

Comment:
Do we know why this patch is changed as part of this commit (the commit updating to the latest 12.9)?